OpenAI-DotNet/Models/ModelsEndpoint.cs

```diff
@@ -83,6 +83,10 @@ public async Task<bool> DeleteFineTuneModelAsync(string modelId)
                 throw new Exception($"Failed to get {modelId} info!");
             }
 
+            if (model.OwnedBy is "openai" or "system" or "openai-dev")
+            {
+                throw new UnauthorizedAccessException($"{model.Id} is not owned by your organization.");
```

Comment:
I didn't add this before just because the api will return error anyway, but also I didn't want to limit it in case someone from open ai actually used this library for some random reason lol.